packages/firebase_ml/test/firebase_ml_test.dart

```diff
@@ -0,0 +1,109 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/services.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:firebase_ml/firebase_ml.dart';
+
+void main() {
+  TestWidgetsFlutterBinding.ensureInitialized();
+  String MODEL_NAME = "myModelName";
+  String MODEL_HASH = "myModelHash";
+  String MODEL_FILE_PATH = "someDestination";
+
+  group('$FirebaseModelManager', () {
+    final List<MethodCall> log = <MethodCall>[];
+    final FirebaseModelManager modelManager = FirebaseModelManager.instance;
+
+    setUp(() {
+      FirebaseModelManager.channel
+          .setMockMethodCallHandler((MethodCall methodCall) async {
+        log.add(methodCall);
+
+        switch (methodCall.method) {
+          case 'FirebaseModelManager#download':
+            var model = <String, String>{};
+            model['modelName'] = MODEL_NAME;
+            model['modelHash'] = MODEL_HASH;
+            return model;
+          case 'FirebaseModelManager#getLatestModelFile':
+            return MODEL_FILE_PATH;
+          case 'FirebaseModelManager#isModelDownloaded':
+            return true;
+          default:
+            return null;
+        }
+      });
+      log.clear();
+    });
+
+    tearDown(() {
+      FirebaseModelManager.channel.setMockMethodCallHandler(null);
+    });
+
+    test('download', () async {
```

Comment:
I noticed 2 of the tests here have the same name 'download'. I'm not sure the best practices for these tests specifically, but we have some guidelines around naming for dart tests. Please take a look here:

https://screenshot.googleplex.com/QxqE2XLAbJv

Reply:
Done